### PR TITLE
OKTA-462650 Set servers parameter to optional ASA /preauthorizations

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** This `servers` is not supported in ASA /preauthorizations. Platform PR 5491 removed the `servers` parameter.
- **Is this PR related to a Monolith release?** Yes, 2022.03.2

### Resolves:

* [OKTA-462650](https://oktainc.atlassian.net/browse/OKTA-462650)

### CODE REVIEW

**Comment by vanngo-okta** [comment]
@phoebeyu-okta, let me know if you want to keep the description: "The Servers that the Preauthorization is valid for (not supported)", or just replace it with "Not supported". Thanks

**Review by phoebeyu-okta — APPROVED** [review]
love it. The change to make this all true will go out on Wednesday, so let's just hold on publishing this until then

File: packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md [hunk 1]
```diff
@@ -1224,7 +1222,7 @@ This endpoint returns a list of objects with the following fields and a `200` co
 | `expires_at`   | string | The Preauthorization ceases to work after the `expires_at` date |
 | `id`   | string | The UUID of the Preauthorization |
 | `projects`   | array | The Projects that the Preauthorization is valid for |
-| `servers`   | array | The Servers that the Preauthorization is valid for |
+| `servers`   | array | Not supported; this field will be removed in a future release of ASA. It is only documented for backward compatibility with legacy clients. |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md:1225]:
```suggestion
| `servers`   | array | Not supported. This field will be removed in a future release of ASA. It's only documented for backward compatibility with legacy clients. |
```

File: packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md [hunk 2]
```diff
@@ -1293,7 +1290,7 @@ This endpoint returns an object with the following fields and a `200` code on a
...
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md:1293]:
```suggestion
| `servers`   | array | Not supported. This field will be removed in a future release of ASA. It's only documented for backward compatibility with legacy clients. |
```

**Review by susanharper-okta — APPROVED** [review]
baby nits. LGTM 🚀 Approving to not hold you up.

**Pull request merged**